readme.md

```diff
@@ -81,7 +92,18 @@ var Panda = function(name, age) {
 ```
 
 Your Answer:
-```text
+```js
+var Panda = function(name, age) {
+  this.name = name
+  this.age = age;
+  this.eat_bamboo
+  this.num_bamboo_eaten = 0
+
+  var eat_bamboo = function()
+  if(eat_bamboo > 0, eat_bamboo++)
+}else {
+      (eat_bamboo)
+}
```

Comment:
This isn't an instance method; it's a private method. Take a look: https://github.com/ga-dc/wdi7-quiz9/blob/solution/readme.md#question-6